model-setup-examples.md

```diff
@@ -0,0 +1,28 @@
+# Model Setup Examples
+
+We have put together some simple examples of how to set up the radio, depending on your needs.
+
+Some of the features we cover are:
+
+* Mixers
+* Special functions
+* Logic Switches
+* Flight modes
+
+OpenTx allows all sorts of interesting ways to program your transmitter.
+
+Hopefully these examples will get you started in the right direction.
+
+All examples are assuming a Taranis X9D+ and using the internal FrSky module in Mode 2.
+
+Because of this, the channel order in the examples with be AETR
```

Comment:
Mode 2 has nothing to do with AETR channel order. It sounds a bit like this. Maybe better say the "Defaults are Mode 2 and Channel AETR", there our example will also use Mode 2 and AETR?